code/script.js

```diff
@@ -1,10 +1,262 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
 
+const nameInput = document.getElementById('name-input')
+
+const inputWrapper = document.getElementById('input-wrapper')
+
+/*const FoodReply = document.getElementById('input-wrapper') ----> hier stimmt etwas nicht */
+
 // Global variables, if you need any, declared here
 
 // Functions declared here
 
+const handleNameInput = (event) => {
+  event.preventDefault()
+  // Store the value in a variable so we can access it after we 
+	// clear it from the input
+  const name = nameInput.value
+  showMessage(name, 'user')
+  nameInput.value = ''
+
+  // After 1 second, show the next option, passing the name into it
+  setTimeout(() => showTipOptions(name), 1000)
+}
+
+
+const showTipOptions = (userName) => {
+  // Post a greeting to the user as the 'bot'
+  showMessage(`Hi ${userName}. <br>
+  <div class="gifhey">
+    <iframe src="https://giphy.com/embed/l0ErUkqRz6Jug7RAI" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/obama-jimmy-fallon-michelle-obama-l0ErUkqRz6Jug7RAI">via GIPHY</a></p>
+  </div>
+    Schön hast du Lust was Neues zu entdecken. Worauf hast du Lust?`, 'bot')
+
+  // Update the input wrapper with the next set of options
+  inputWrapper.innerHTML = `
+    <button id="videoBtn">Videos</button>
+    <button id="buecherBtn">Bücher</button>
+    <button id="podcastBtn">Podcasts</button>
+  `
+
+  // Next step here - add event listeners to the 3 new buttons we added above
+  document.getElementById('videoBtn').addEventListener('click', handleVideoInput);
+  document.getElementById('buecherBtn').addEventListener('click', handleBuecherInput);  
+  document.getElementById('podcastBtn').addEventListener('click', handlePodcastInput);
+}
+
+
+const handleVideoInput = () => { 
+  showMessage(`Videos`, 'user')
+
+  setTimeout(() => showVideoOptions(), 1000)
+
+}
+
+
+const showVideoOptions = () => {
+  showMessage(`Du willst Videos? Videos sollst du kriegen. In welche Richtung soll es gehen?`, 'bot') 
+
+  inputWrapper.innerHTML = `
+  <button id="gleichberechtigungBtn">Gleichberechtigung & Empowerment</button>
+  <button id="techBtn">Future & Tech</button>
+  <button id="popculture">Pop Culture</button>
```

Comment:
The reason the popculture button is not working is that you have different id's. Here it just says "popculture" but on row 67 it says 'popcultureBtn'